File: Android/Muse/src/main/java/com/tripath/muse/activity/SplashScreenActivity.java
/*
 *  SplashScreenActivity.java 1.0.0 2017/10/23
 *
 * Copyright (c) 2017 RyoRyeong KIM
 * All rights reserved
 *
 * this software is the confidential and proprietary information to Ryo Ryeong KIM
 * you shall not disclose such Confidential Information and shall use it only in
 * accordance with the terms of the license agreement you entered into with RyoRyeong KiM
 */

package com.tripath.muse.activity;

import android.content.Intent;
import android.os.Bundle;

import com.tripath.muse.MainActivity;
import com.tripath.muse.R;

/**
 * 진입화면
 *
 * @version  1.0.0 23 Oct 2017
 * @author Kim RyoRyeong
 */
public class SplashScreenActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_splash_screen);

        baseHandler.postDelayed(
                new Runnable() {
                    @Override
                    public void run() {

                        Intent intent = new Intent(SplashScreenActivity.this , MainActivity.class);
                        intent.setFlags( Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TOP );
                        overridePendingTransition( 0, android.R.anim.slide_out_right );
                        startActivity(intent);

                        finish();
                    }
                },
                1000
        );
    }

}

File: Android/KotlinRepository/QSalesPrototypeKotilnVersion/main/java/tripath/com/samplekapp/SplashScreenActivity.kt
package tripath.com.samplekapp

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import android.widget.Toast

/**
 *  !! 란 ?
 */
class SplashScreenActivity  : BaseActivity() , View.OnClickListener {
    override fun onClick(p0: View?) {
        when(p0!!.id){
            R.id.button ->{
                var intent = Intent(applicationContext,MainActivity::class.java )
                startActivity( intent )
            }
            else -> {
                Toast.makeText(applicationContext,"unexpeded error",Toast.LENGTH_SHORT).show();
            }
        }
    }


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash_screen)


        baseHandler.postDelayed(
            object : Runnable {
                override fun run() {
//                    var intent = Intent(this@SplashScreenActivity,MainActivity::class.java )
//                    startActivity( intent )

                    redirectMainActivityWithAnim(this@SplashScreenActivity)
                }
            },1000)

        findViewById( R.id.button).visibility = View.GONE
        findViewById( R.id.button).setOnClickListener(this )

    }


}
